lib/cloudantBaseService.ts

```diff
@@ -151,6 +155,7 @@ export abstract class CloudantBaseService extends BaseService {
       (auth as CouchdbSessionAuthenticator).configure(this.baseOptions);
     }
   }
+
```

Comment:
I would like to keep this new line, since this way the code will comply with other function description formats.
Every function has a comment before it without new lines and every comment is preceded with a new line, after the previous function is closed.

Reply:
Ok 👍